package.json

```diff
@@ -48,8 +48,8 @@
     "mongodb": "4.10.0",
     "mustache": "4.2.0",
     "parse": "3.4.2",
-    "pg-monitor": "1.5.0",
-    "pg-promise": "10.12.1",
+    "pg-monitor": "2.0.0",
+    "pg-promise": "11.0.0",
```

Comment:
Can you run `npm install` to update the package-lock.json?

Reply:
done